src/common/com/intellij/plugins/haxe/ide/annotator/HaxeSemanticAnnotator.java

```diff
@@ -49,11 +51,11 @@
 
 public class HaxeSemanticAnnotator implements Annotator {
   @Override
-  public void annotate(PsiElement element, AnnotationHolder holder) {
+  public void annotate(@NotNull PsiElement element, @NotNull AnnotationHolder holder) {
     analyzeSingle(element, holder);
   }
 
-  static void analyzeSingle(final PsiElement element, AnnotationHolder holder) {
+  private static void analyzeSingle(final PsiElement element, AnnotationHolder holder) {
```

Comment:
Question: Why the "private" everywhere?  Is there a reason to hide static functions?  You don't have to change anything, here. I'm just curious if there is a performance gain or just personal preference on your part.